src/main/java/sirius/biz/model/MD5ApiTokenHashFunction.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.model;
+
+import com.google.common.base.Charsets;
+import com.google.common.hash.Hashing;
+import sirius.kernel.di.std.Register;
+
+/**
+ * Uses md5 hashing to hash the apiToken.
+ */
+@Register
+public class MD5ApiTokenHashFunction implements ApiTokenHashFunction {
```

Comment:
where's the difference to LegacyMD5ApiTokenHashFunction - can't we make that a protected method in the usermanager so that it can be overwritten on demand